### PR TITLE
test pulls for AL only

### LINKED ISSUE #1: create starter subset query
Before we get to the NAWQA test pull, we should test on an even small test pull - e.g., 2 tiny states, 2 constituents, 6 months of data. Adjust the cfg files to specify this smaller pull

### PR DESCRIPTION
**Pull request by lindsaycarr**
Fixes #1.

Only thing I'm not sure about is how the following code happens twice. I want to verify that is correct before I go try this out for all states.
```
2018-05-22 14:09:08: getting inventory for temperature
2018-05-22 14:10:11: getting inventory for phosphorus
2018-05-22 14:11:43: getting inventory for nitrate
```

-----

Console output:

```
> scmake(remake_file = "remake.yml")
Starting build at 2018-05-22 14:09:08
<  MAKE > all
[    OK ] wqp_state_codes
[    OK ] 1_wqpdata/tmp/data
[    OK ] 1_wqpdata/out/inventory
[    OK ] 1_wqpdata/out/data
[    OK ] 1_wqpdata/log
[    OK ] lib/cfg
[    OK ] wqp_states
[ BUILD ] wqp_codes                                         |  wqp_codes <- yaml.load_file("1_wqpdat...
[    OK ] wqp_pull
[    OK ] wqp_pull_folders
[ BUILD ] 1_wqpdata/out/inventory/wqp_inventory.feather.ind |  inventory_wqp(ind_file = "1_wqpdata/o...
2018-05-22 14:09:08: getting inventory for temperature
2018-05-22 14:10:11: getting inventory for phosphorus
2018-05-22 14:11:43: getting inventory for nitrate
sample inventory complete, required 222 seconds
2018-05-22 14:12:50: getting additional site data
site inventory complete, required 39 seconds
[ BUILD ] wqp_pull_partitions                               |  wqp_pull_partitions <- partition_inve...
[ BUILD ] wqp_pull_plan                                     |  wqp_pull_plan <- plan_wqp_pull(partit...
[ BUILD ] tasks_1_wqp.yml                                   |  create_wqp_pull_makefile(makefile = "...
run all tasks with
1_wqpdata/log/tasks_1_wqp.ind:
  command: make(I('tasks_1_wqp'), remake_file='tasks_1_wqp.yml')
[ BUILD ] 1_wqpdata/log/tasks_1_wqp.ind                     |  loop_tasks(task_plan = wqp_pull_plan,...

### Starting loop attempt 1 of 30 with 3 tasks remaining:
Building task 1 of 3 in loop: 1_wqpdata/out/data/Alabama_phosphorus_001.feather.ind (#2 of 4 total)
2018-05-22 14:13:40: getting inventory for temperature
2018-05-22 14:14:43: getting inventory for phosphorus
2018-05-22 14:15:41: getting inventory for nitrate
sample inventory complete, required 173 seconds
2018-05-22 14:16:33: getting additional site data
site inventory complete, required 32 seconds
WQP pull for Alabama_phosphorus_001 took 133.310000000001 seconds and returned 49892 rows
Building task 2 of 3 in loop: 1_wqpdata/out/data/Alabama_temperature_001.feather.ind (#3 of 4 total)
WQP pull for Alabama_temperature_001 took 394.33 seconds and returned 132668 rows
Building task 3 of 3 in loop: 1_wqpdata/out/data/Alabama_temperature_002.feather.ind (#4 of 4 total)
WQP pull for Alabama_temperature_002 took 382.640000000001 seconds and returned 132809 rows

### Final check for completeness of all targets
Starting build at 2018-05-22 14:34:59
<  MAKE > tasks_1_wqp
[    OK ] wqp_state_codes
[    OK ] 1_wqpdata/out/inventory
[    OK ] wq_dates
[    OK ] wqp_states
[    OK ] wqp_codes
[    OK ] wqp_pull
[    OK ] 1_wqpdata/out/inventory/wqp_inventory.feather.ind
[    OK ] wqp_pull_partitions
[ BUILD ] partition_Alabama_nitrate_001                          |  partition_Al...
[    OK ] 1_wqpdata/tmp/data/Alabama_nitrate_001.feather.ind
[    OK ] 1_wqpdata/out/data/Alabama_nitrate_001.feather.ind
[    OK ] partition_Alabama_phosphorus_001
[    OK ] 1_wqpdata/tmp/data/Alabama_phosphorus_001.feather.ind
[    OK ] 1_wqpdata/out/data/Alabama_phosphorus_001.feather.ind
[    OK ] partition_Alabama_temperature_001
[    OK ] 1_wqpdata/tmp/data/Alabama_temperature_001.feather.ind
[    OK ] 1_wqpdata/out/data/Alabama_temperature_001.feather.ind
[    OK ] partition_Alabama_temperature_002
[    OK ] 1_wqpdata/tmp/data/Alabama_temperature_002.feather.ind
[    OK ] 1_wqpdata/out/data/Alabama_temperature_002.feather.ind
[ BUILD ] tasks_1_wqp                                            |  tasks_1_wqp ...
Finished build at 2018-05-22 14:34:59
Build completed in 0.00 minutes
[    OK ] lib/cfg/gd_config.yml
[ ----- ] 1_wqpdata
[ ----- ] all
Finished build at 2018-05-22 14:35:00
Build completed in 25.86 minutes
```

### CODE REVIEW

File: build/status/MV93cXBkYXRhL291dC9pbnZlbnRvcnkvd3FwX2ludmVudG9yeS5mZWF0aGVyLmluZA.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: 0.3.0
+name: 1_wqpdata/out/inventory/wqp_inventory.feather.ind
+type: file
+hash: 99b641c3ad4e5963d5b094606069cb98
+time: 2018-05-22 19:17:09 UTC
+depends:
+  wqp_state_codes: a931f55b406b1486a4b326c73b1c0e33
+  wqp_states: ccdbb22fc8750e99996090cb41485918
+  wqp_codes: b719e8f0d49ca909602bf52f83cb9bdf
+  1_wqpdata/out/inventory: .na.character
```
[the remark this answers is not in the document]
Comment (aappling-usgs) [reply build/status/MV93cXBkYXRhL291dC9pbnZlbnRvcnkvd3FwX2ludmVudG9yeS5mZWF0aGVyLmluZA.yml]: I bet we need to wrap `'1_wqpdata/out/inventory'` in `I()` in the call to `inventory_wqp` to get this not to happen.

**Comment by aappling-usgs** [comment]
Added a few comments above that I hope you'll read, but I don't think those changes (if any) need to go into this PR. Keep on keepin' on.

File: build/status/MV93cXBkYXRhL291dC9pbnZlbnRvcnkvd3FwX2ludmVudG9yeS5mZWF0aGVyLmluZA.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (lindsaycarr) [reply build/status/MV93cXBkYXRhL291dC9pbnZlbnRvcnkvd3FwX2ludmVudG9yeS5mZWF0aGVyLmluZA.yml]: Are you talking about this line? https://github.com/USGS-R/nawqa_wqp/blob/master/1_wqpdata.yml#L71

[the remark this answers is not in the document]
Comment (aappling-usgs) [reply build/status/MV93cXBkYXRhL291dC9pbnZlbnRvcnkvd3FwX2ludmVudG9yeS5mZWF0aGVyLmluZA.yml]: Oh, hmm, yes i was but i hadn't located the line yet. We might need to turn it into a function argument to be able to wrap it in `I()` - I haven't tried using `I()` for `depends:` items

[the remark this answers is not in the document]
Comment (aappling-usgs) [reply build/status/MV93cXBkYXRhL291dC9pbnZlbnRvcnkvd3FwX2ludmVudG9yeS5mZWF0aGVyLmluZA.yml]: I doubt it, but nothing is impossible. What's the default `mock_get` value in `gd_put` in your version of scipiper? And what's your version of scipiper?

[the remark this answers is not in the document]
Comment (aappling-usgs) [reply build/status/MV93cXBkYXRhL291dC9pbnZlbnRvcnkvd3FwX2ludmVudG9yeS5mZWF0aGVyLmluZA.yml]: I just pulled and tried to build the inventory directly a couple of times. scipiper saw no need to rebuild it at all.
```r
> scmake('1_wqpdata/out/inventory/wqp_inventory.feather.ind','1_wqpdata.yml')
Starting build at 2018-05-23 07:52:43
<  MAKE > 1_wqpdata/out/inventory/wqp_inventory.feather.ind
[ BUILD ] wqp_state_codes                                   |  wqp_state_codes <- get_wqp_state_codes()
[  READ ]                                                   |  # loading packages
[ BUILD ] 1_wqpdata/out/inventory                           |  dir.create("1_wqpdata/out/inventory", recursive = TRUE, showWarni...
[ BUILD ] wqp_states                                        |  wqp_states <- yaml.load_file("1_wqpdata/cfg/wqp_states.yml")
[ BUILD ] wqp_codes                                         |  wqp_codes <- yaml.load_file("1_wqpdata/cfg/wqp_codes.yml")
[    OK ] 1_wqpdata/out/inventory/wqp_inventory.feather.ind
Finished build at 2018-05-23 07:52:44
Build completed in 0.03 minutes
Warning message:
package ‘yaml’ was built under R version 3.4.4 
> scmake('1_wqpdata/out/inventory/wqp_inventory.feather.ind','1_wqpdata.yml')
Starting build at 2018-05-23 07:52:47
<  MAKE > 1_wqpdata/out/inventory/wqp_inventory.feather.ind
[    OK ] wqp_state_codes
[    OK ] 1_wqpdata/out/inventory
[    OK ] wqp_states
[    OK ] wqp_codes
[    OK ] 1_wqpdata/out/inventory/wqp_inventory.feather.ind
Finished build at 2018-05-23 07:52:47
Build completed in 0.00 minutes
```
that's what we want, i think, but also offers no insight into why it ran twice for you.

**Pull request merged**